frappe/desk/doctype/note/note.js

```diff
@@ -1,43 +1,6 @@
-frappe.ui.form.on("Note", {
-	refresh: function (frm) {
```

Comment:
This "not editable" by default was a "feature". It makes reading easier :sweat_smile: